Expecto/Expecto.fs

```diff
@@ -643,3 +643,44 @@ module Tests =
   /// Returns 0 if all tests passed, otherwise 1
   let runTestsInAssemblyWithCLIArgs cliArgs args =
     runTestsInAssemblyWithCLIArgsAndCancel CancellationToken.None cliArgs args
+
+  let runTestsReturnLogs cliArgs args tests =
+    let tryBuildConfig cliArgs args =
+        let config =
+          Seq.fold (fun s a -> foldCLIArgumentToConfig a s)
+            ExpectoConfig.defaultConfig cliArgs
+
+        match ExpectoConfig.fillFromArgs config args with
```

Comment:
I believe this results in double args parsing, but it seemed like the least complex option (and easiest to follow) and we're not doing that much parsing generally. It also only applies to this interactive run method